superset-frontend/src/explore/components/controls/AnnotationLayerControl/index.tsx

```diff
@@ -124,7 +159,11 @@ class AnnotationLayerControl extends React.PureComponent {
     this.props.onChange(annotations);
   }
 
-  renderPopover(popoverKey, annotation, error) {
+  renderPopover = (
+    popoverKey: number | string,
+    annotation: Annotation,
```

Comment:
According to the following statement (i.e. `annotation?.name`), this must be optional.
```suggestion
    annotation: Annotation | undefined,
```